bubble_sort.rb

```diff
@@ -0,0 +1,58 @@
+integer_arr = [4, 3, 78, 2, 0, 2]
+string_arr = %w[hi hello hey]
+
+# Bubble sort
+def bubble_sort(array)
+  times_loop = array.length - 1
+  # Iterate trhough the array a number of times to sort it
+  times_loop.times do
+    # Swap counter to check if array is already sorted
+    swap_counter = 0
+    # Compare every item in array to the one next to it
+    array.each_with_index do |item, index|
+      # Check if the a number is greater than the b next number
+      a = item
+      b = array[index + 1]
+      is_greater = a <=> b
+      next unless is_greater == 1
+
+      array[index + 1] = a
+      array[index] = b
+      swap_counter += 1
+    end
+    # Stop the loop if array is sorted
+    break if swap_counter.zero?
+  end
+  yield
+end
+
```

Comment:
## Bubble_sort
- This function should return an array. Follow this [example](https://github.com/bertil291utn/bubble-sort-project/blob/master/bubble_sort_algorithm.rb)
- Then try to check this test
```ruby
unsorted = (1..10).to_a.reverse!
bubble_sort(unsorted))
# should be equal to  
unsorted.sort
```